avm/ptn/dev-ops/cicd-agents-and-runners/tests/e2e/aca-aci/main.test.bicep

```diff
@@ -0,0 +1,82 @@
+targetScope = 'subscription'
+
+metadata name = 'Using only defaults for Azure DevOps self-hosted agents using both Azure Container Instances and Azure Container Apps.'
+metadata description = 'This instance deploys the module with the minimum set of required parameters for Azure DevOps self-hosted agents in Azure Container Instances and Azure Container Apps.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-devopsrunners-${serviceShort}-rg'
+
+#disable-next-line no-hardcoded-location // Due to quotas and capacity challenges, this region must be used in the AVM testing subscription
+var enforcedLocation = 'eastus2'
+
+@description('Required. The name of the Azure DevOps agents pool.')
+param agentsPoolName string = 'agents-pool'
+
+@description('Required. The name of the Azure DevOps organization.')
+param devOpsOrganization string = 'azureDevOpsOrganization'
+
+@description('Required. The personal access token for the Azure DevOps organization.')
+@secure()
+param personalAccessToken string = newGuid()
```

Comment:
Should make use of the ['Custom CI secrets'](https://github.com/Azure/Azure-Verified-Modules/blob/392686160d3c227feff50d314d1f1080f7d45379/docs/content/contributing/bicep/bicep-contribution-flow/custom-ci-secrets.md) feature once https://github.com/Azure/bicep-registry-modules/pull/3149 is merged

Reply:
Applies also to other tests

Reply:
In the tests I'm using dummy values just to validate that the deployment is successful, I'm not testing that a GitHub workflow or an Azure DevOps pipeline can run on those self-hosted agents/runners given the complexity of managing this (renewing PATs every 3-4 days,..etc) does it make sense to keep it a random GUID ?

Reply:
Ah, I see. Well in that case nevermind :) It looked like you're requiring an actual environment